site/authors/justinharringa.adoc

```diff
@@ -1,9 +1,16 @@
 = About the Author
 :page-layout: author
 :page-author_name: Justin Harringa
-:page-linkedin: justinharringa
 :page-github: justinharringa
-:page-twitter: justinharringa
 :page-authoravatar: ../../images/images/avatars/justinharringa.jpeg
+:page-twitter: justinharringa
+:page-linkedin: justinharringa
 
-A user of Jenkins since the early Hudson days, Justin has contributed to various plugins (including the link:https://plugins.jenkins.io/chatter-notifier[Chatter Notifier Plugin]) and has built a few other plugins such as the recently released link:https://plugins.jenkins.io/config-driven-pipeline[Config-Driven Pipeline plugin] as well as the link:https://plugins.jenkins.io/argus-notifier[Argus Notifier plugin].  He's used Jenkins in companies such as link:https://www.deere.com[John Deere] and link:https://salesforce.com[Salesforce] as well as during his time in Brazil at link:https://www.cesar.org.br[C.E.S.A.R.] for some of their international clients. Justin enjoys software development, CI/CD, travel, music, and learning.
+A user of Jenkins since the early Hudson days, Justin has contributed to various plugins (including the
+link:https://plugins.jenkins.io/chatter-notifier[Chatter Notifier Plugin]) and has built a few other plugins
+such as the recently released link:https://plugins.jenkins.io/config-driven-pipeline[Config-Driven Pipeline plugin]
+as well as the https://plugins.jenkins.io/argus-notifier[Argus Notifier plugin]. 
+He's used Jenkins in companies such as link:https://www.deere.com[John Deere] and 
+link:https://salesforce.com[Salesforce] as well as during his time in Brazil at 
+link:https://www.cesar.org.br[C.E.S.A.R.] for some of their international clients. Justin enjoys software development, 
+CI/CD, travel, music, and learning.
```

Comment:
```suggestion
A user of Jenkins since the early Hudson days, Justin has contributed to various plugins (including the link:https://plugins.jenkins.io/chatter-notifier[Chatter Notifier Plugin]) and has built a few other plugins such as the recently released link:https://plugins.jenkins.io/config-driven-pipeline[Config-Driven Pipeline plugin] as well as the https://plugins.jenkins.io/argus-notifier[Argus Notifier plugin]. 
He's used Jenkins in companies such as link:https://www.deere.com[John Deere] and link:https://salesforce.com[Salesforce] as well as during his time in Brazil at link:https://www.cesar.org.br[C.E.S.A.R.] for some of their international clients. 
Justin enjoys software development, CI/CD, travel, music, and learning.
```